responses/lv/HassRespond.yaml

```
@@ -0,0 +1,10 @@
+language: lv
+responses:
+  intents:
+    HassRespond:
+      default: ""
+      hello: "Sveicieni no Home Assistant."
+      listening: "Nē, Es klausos tikai tad, kad tu pasaki modināšanas frāzi."
+      data: "Jūsu dati tiek nosūtīti uz Home Assistant serveri."
+      commands: "Lai uzzinātu ko jūs varat man jautāt, apmeklējiet H.A. punkts I.O. slīpsvītra voice."
+      creator: "Mani izveidoja brīnišķīgā Home Assistant komūna, sastavoša no domātājiem no visas pasaules."
```

```suggestion
      creator: "Mani izveidoja brīnišķīgā Home Assistant kopiena, kas sastāv no domātājiem visā pasaulē."
```
What do you think of this?

---

Actualy I do not like nether :). But could not get better translation. Anyways yours sound bit better.